### DISCUSSION TITLE
How to run templates of private github repo using SDK v3

### DISCUSSION THREAD

**Discussion by VladimirFilipovic** [projectdiscovery/nuclei-templates#8761] [Q&A]
Hello everyone, 

How can i download/run templates of my private github repository using SDK?
I've read in docs that it can be done using cli tool. I'm interested if that is possible using the go library?

**Comment by MetzinAround** [comment]
Hi! I think we answered this question in the discord, but here's the link for others who see this issue and have a similar question: https://pkg.go.dev/github.com/projectdiscovery/nuclei/v3/lib
